### ISSUE TITLE
Nuget and dependency recommendations

### ISSUE THREAD

**Issue by mshindal** [Microsoft/FFmpegInterop#58]
Any chance this will get published to Nuget? I suspect not, since building FFmpeg is such a pain. If not, would the "cleanest" way of adding this to my project be adding it as a git submodule?

**Comment by jakubsuchybio** [comment]
+1 on nuget

**Comment by NickDarvey** [comment]
+1

**Comment by scabana** [comment]
Yup, that would be very very nice. I would like not to setup a build machine to be able to build FFmpeg!
